src/sinks/new_relic_logs.rs

```diff
@@ -115,9 +119,8 @@ impl NewRelicLogsConfig {
 
         let batch = self.batch.use_size_as_bytes()?;
         let batch = BatchConfig {
-            // The max request size is 10MiB, so in order to be comfortably
-            // within this we batch up to 5MiB.
-            max_bytes: Some(batch.max_bytes.unwrap_or(bytesize::mib(5u64) as usize)),
+            // New Relic Logs API accepts payloads only up to 1MB (10⁶ bytes)
```

Comment:
Did you mean to put `10MB` here?

Reply:
I see you did not get distracted by the fancy ⁶ character.

Reply:
:smile: its presence actually inspired me to figure out how to type superscript via [compose key](http://www.kbdedit.com/manual/ex17_compose_key_chained_dead.html).

Reply:
I meant to put `1MB`, because this is the actual limit. I will also reformat the fancy character to something less fancy.

Reply:
Oops, my apologies, I was thinking it was supposed to be 10MiB. I do see the 1MiB documented limit. I'm confused what limit of 10 MiB that old comment was referring to.